### PR TITLE
VAL-7830 Enable Authorization for Azure event grid events

### PR DESCRIPTION
**Pull request by jayeshvaltix**

### CODE REVIEW

**Review by pafrank-mcd — APPROVED** [review]
lgtm

File: azure-setup.sh [hunk 1]
```diff
@@ -229,15 +229,17 @@ if [ "$terms_rsp" != "true" ]; then
...
+echo "Creating event subscription for an Azure subscription, with enabled authorization"
 az eventgrid event-subscription create \
     --source-resource-id "/subscriptions/${sub_id}" \
     --name "$EVENT_SUB_NAME" \
     --endpoint "$webhook_endpoint" \
     --included-event-types \
     Microsoft.Resources.ResourceWriteSuccess \
     Microsoft.Resources.ResourceDeleteSuccess \
-    Microsoft.Resources.ResourceActionSuccess
+    Microsoft.Resources.ResourceActionSuccess \
```
Comment (pashracisco) [inline azure-setup.sh:240]: Don't we need --endpoint-type webhook?

**Review by ganeshvaltix — APPROVED** [review]
lgtm

File: azure-setup.sh (hunk 1 above)
Comment (jayeshvaltix) [reply azure-setup.sh:240]: --endpoint-type   : The type of the destination endpoint. Default: webhook.

**Pull request merged**